cmd/nodecmd/create_aws.go

```diff
@@ -229,30 +230,32 @@ func createAWSInstance(ec2Svc *ec2.EC2, region, ami string, usr *user.User) (Clo
 		} else {
 			ux.Logger.PrintToUser("Failed to create AWS cloud server")
 		}
-		// we stop created instances so that user doesn't pay for unused EC2 instances
-		instanceIDs, instanceIDErr := terraformaws.GetInstanceIDs(app.GetTerraformDir())
-		if instanceIDErr != nil {
-			return CloudConfig{}, instanceIDErr
-		}
-		failedNodes := []string{}
-		nodeErrors := []error{}
-		for _, instanceID := range instanceIDs {
-			ux.Logger.PrintToUser(fmt.Sprintf("Stopping AWS cloud server %s...", instanceID))
-			if stopErr := awsAPI.StopInstance(ec2Svc, instanceID, "", false); stopErr != nil {
-				failedNodes = append(failedNodes, instanceID)
-				nodeErrors = append(nodeErrors, stopErr)
+		if strings.Contains(err.Error(), constants.ErrCreatingAWSNode) {
+			// we stop created instances so that user doesn't pay for unused EC2 instances
```

Comment:
same here. print a msg of what is happening and probably encapsulate this code together with the stop command code